plugins/modules/system/listen_ports_facts.py

```diff
@@ -155,41 +155,59 @@
 from ansible.module_utils.basic import AnsibleModule
 
 
+def split_pid_name(pid_name):
+    try:
+        pid, name = pid_name.split("/")
+    except ValueError:
+        return 0, ""
```

Comment:
Since the other branch returns two strings, I think this one should too:
```suggestion
        return "0", ""
```
Alternatively, how about returning `int(pid)` here?

Reply:
Correct both should return the same format. Went with `int, string` tuple. 
Keeping the second int cast in the result to have it uniform with the `ss` function and easier to see which entries are int.